content/scout/guides/vex.md

```diff
@@ -0,0 +1,345 @@
+about how a software product is affected by a vulnerable component.
+
+The concept of VEX is defined by a working group by
+the United States Cybersecurity and Infrastructure Security Agency (CISA).
+At the core of VEX are exploitability assessments.
+These assessments describe the status of a given CVE for a product.
+The possible vulnerability statuses in VEX are:
+
+- Not affected: No remediation is required regarding this vulnerability.
+- Affected: Actions are recommended to remediate or address this vulnerability.
+- Fixed: These product versions contain a fix for the vulnerability.
+- Under investigation: It is not yet known whether these product versions are affected by the vulnerability. An update will be provided in a later release.
+
+There are multiple implementations of VEX,
+and the exact format and properties of a VEX document differ
+depending on the implementation of VEX that you use.
+Examples of VEX implementations include:
+
+- OpenVEX
+- Open Security Advisory Framework (OSAF)
+- CycloneDX
+- SPDX
+
+This guide uses the OpenVEX implementation in its examples.
+For more information about OpenVEX,
+refer to the [specification](https://github.com/openvex/spec).
+
+In all implementations, the core idea is the same:
+to provide a framework for describing the impact of vulnerabilities.
+Key components of VEX regardless of implementation includes:
+
+VEX document
+: A type of security advisory for storing VEX statements.
+  The format of the document depends on the specific implementation.
+
+VEX statement
+: Describes the status of a vulnerability in a product,
+  whether it's exploitable, and whether there are ways to remediate the issue.
+
+Justification and impact
+: Depending on the vulnerability status, statements include a justification
+  or impact statement describing why a product is or isn't affected.
+
+Action statements
+: Describe how to remediate or mitigate the vulnerability.
+
+This added context that VEX provides around vulnerabilities
+helps organizations perform risk assessment of software products.
+
+## Create and enable a repository
+
+To get started, create a sample project to work with.
+Use the [Docker Scout demo service](https://github.com/docker/scout-demo-service)
+template repository to bootstrap a new repository in your own GitHub organization.
+
+1. Create the repository from the template.
+2. Clone the Git repository to your machine.
+3. Build an image from the repository and push it to a new Docker Hub repository.
+
+   ```console
+   $ cd scout-demo-service
+   $ docker build --provenance=true --sbom=true --tag <ORG>/scout-demo-service:v1 --push .
+   ```
+
+4. Enable Docker Scout on the repository.
+
+   ```console
+   $ docker scout repo enable <ORG>/scout-demo-vex:v1
+   ```
+
+## Inspect vulnerability
+
+Use the `docker scout cves` command to view the vulnerabilities for the image.
+For the purpose of this guide, we'll concentrate on a particular CVE: CVE-2022-24999.
```

Comment:
Would it be useful to link out to the CVE details?

Reply:
Not in this case, the CVE in question is not important